### DISCUSSION TITLE
Kata Módulo 10

### DISCUSSION THREAD

**Discussion by vicogarcia16** [LaunchX-InnovaccionVirtual/CursoIntroPython#203] [Q&A] [answered]
Mi pregunta: el archivo Módulo10Katas.ipynb ¿Será publicado? Solo esta la información sobre el tema

**Comment by AzaelBarbosa** [comment]
@vicogarcia16 En el mismo archivo especifica que debes hacer los ejercicios que vienen en el mismo Documento, es decir ya sea que implementes un ejercicio propio o bien hagas los mismos ejemplos que explican en el archivo.

![image](https://user-images.githubusercontent.com/85709470/153725738-b11113c5-3b79-4830-94c7-9e77d508bad9.png)

**Comment by DanielOe21** [comment, marked as answer]
# Kata 10 = Kata 2
Hola @vicogarcia16 ! El modulo 10 Es un archivo de solo texto (Markdown). La kata es replicar los ejemplos que vienen ahí, haz de cuenta q es la misma polémica de la Kata 2 😅

Puedes crear un archivo de texto Markdown con capturas de pantalla, o puedes intentar con una notebook de Jupyter si hay partes que ocupan ejecutar código como en las demás katas.

Espero haberte ayudado, por favor no olvides marcar tu pregunta como CONTESTADA si te parece que ha sido así.

### Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Reply by DiegoPenalozaJuarez** [reply]
Una pregunta @DanielOe-21, sabes tu como agregar esas capturas de pantalla en un archivo de texto (Markdown) en Visual Studio? Lo intente con la Kata 2 usando un archivo .md en VS, guardaba las capturas de pantalla en mi equipo y desde el notebook use esto que encontre en internet: 

`![title](Nombre_imagen.png)`

Pero no logre lo que buscaba y termine haciendo un pdf. Tu sabes como se realiza? 

Gracias

**Reply by DanielOe21** [reply]
@DiegoPenalozaJuarez en la parte de 'Nombre de la imagen'  se pone la ruta del archivo, en este caso lo que yo acostumbro a hacer sería poner una carpeta ('resources') dentro del directorio en el que está mi archivo .md y ahi guardar las imágenes, posteriormente en el .md direccionar la ruta de la imagen a algo tipo (./resources/nombre-imagen.png). 

No lo he intentado directamente en un markdown .md, trabajo más con notebooks de jupyter, pero debería resultar 😋

**Reply by AngelCruzO** [reply]
@DiegoPenalozaJuarez Si instalas la extensión "Office Viewer(Markdown Editor", puedes incluir tus imágenes como si fuera un archivo d word
